sheetshuttle/sheet_collector.py

```diff
@@ -286,12 +294,19 @@ def get_region(self, region_name: str):
         Returns:
             Region: the region object from the self.regions dictionary
         """
-        requested_region: Region = self.regions[region_name]
-        return requested_region
+        # update this with the new indexing structure
+        # requested_region: Region = self.regions[region_name]
+        # return requested_region
+        # New Approach
+        requested_tab: Tab = self.tabs[sheet[region_name]]
+        print(requested_tab)
+        return requested_tab
 
     def print_sheet(self):
         """Iterate through self.regions and print the contents."""
-        for region_id, region in self.regions.items():
+        # update this with the new indexing structure
+        # Fixed indexing structure in this function
+        for region_id, region in self.tabs[sheet[regions.items()]]:
```

Comment:
what is `sheet` referring to here? This variable doesn't look to be defined in this scope